scripts/gceworker.sh

```diff
@@ -54,6 +55,22 @@ case ${1-} in
     shift
     gcloud compute ssh "${name}" "$@"
     ;;
+    mount)
+    shift
+    if [ $# != 2 ]; then
+      echo "usage: $0 <remote-path> <local-path>"
```

Comment:
helpful defaults here? Could use `${2-${GOPATH}/src/github.com/cockroachdb/cockroach}` and wouldn't you by default mount it over the same location on the other end?